snapshots/VaultRouter.json

```diff
@@ -1,7 +1,7 @@
 {
-  "claimDeposit": "1153430",
+  "claimDeposit": "1104497",
```

Comment:
This comes from remove the `wire()` method in tests

Reply:
Want to double-check this, in case we have some issue in the Gateway/Multiadapters. 

Seems somehow that adding adapters and overwritting them later, increase the computation of parsing messages later :think

Reply:
Found the issue, nothing to worry about IIUC. This code was computed previously: https://github.com/centrifuge/protocol-v3/blob/main/src/common/adapters/MultiAdapter.sol#L122-L126

But everything works as expected, we're just delete an already deleted storage